upload.prx.org/upload.prx.org.yml

```diff
@@ -83,3 +113,22 @@ Resources:
       Role: !GetAtt LambdaBasicExecutionIAMRole.Arn
       Runtime: nodejs4.3
       Timeout: 5
+  UploadErrorAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmName: "Upload lambda errors"
+      AlarmActions:
+        - !Ref OpsDebugMessagesSNSTopicArn
```

Comment:
Probably we want these alarms to go to a more important SNS topic. Maybe **warn** for this one?